client/option.go

```diff
@@ -38,6 +40,43 @@ func (f ObkvOptionFunc) apply(opts *ObkvOptions) {
 	f(opts)
 }
 
+type ObkvQueryOption interface {
+	apply(opts *ObkvQueryOptions)
+}
+type ObkvQueryOptionFunc func(opts *ObkvQueryOptions)
+
+func NewObkvQueryOption() *ObkvQueryOptions {
+	return &ObkvQueryOptions{
+		queryFilter:   nil,
+		hTableFilter:  nil,
+		selectColumns: nil,
+		indexName:     "",
+		batchSize:     -1,
+		maxResultSize: -1,
+		limit:         -1,
+		offset:        0,
+		scanOrder:     table.Forward,
+		isHbaseQuery:  false,
+	}
+}
+
+type ObkvQueryOptions struct {
+	queryFilter   interface{}
```

Comment:
why queryFilter and hTableFilter are interface{}

Reply:
queryFilter and hTableFilter will be implemented later.